src/Microsoft.XmlSerializer.Generator/pkg/Microsoft.XmlSerializer.Generator.pkgproj

```diff
@@ -5,6 +5,8 @@
     <ProjectReference Include="..\src\Microsoft.XmlSerializer.Generator.csproj" />
   </ItemGroup>
   <PropertyGroup>
+    <ReleaseNotes>https://go.microsoft.com/fwlink/?LinkID=863421</ReleaseNotes>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=858594</projectUrl>
```

Comment:
Should the first `p` be capital?

Reply:
I will update to use capital P. But from https://docs.microsoft.com/en-us/nuget/schema/nuspec, it seems using lower p. Both should work.